inspec/google_compute_zone.rb

```diff
@@ -47,8 +51,15 @@ def url
     'projects/{{project}}/zones/{{name}}'
   end
 
-  # TODO
-  def parse end
+  def parse
```

Comment:
Will parse ever be called if `@fetched == nil`? This entire method will crash with the least intuitive error (`[] not found on NilClass`?) if it does.

Reply:
Nope, the initialize method (when it is included) checks that fetched is not nil prior to calling this.